### PR TITLE
chore(data-binding):added note for ValueExpression

### PR DESCRIPTION
**Pull request by ntacheva**
Added note for ValueExpression and linked the corresponding kb - http://localhost/blazor-ui/knowledge-base/requires-valueexpression.

Based on customer feedback for this article:
23 Jun 2021 7:43 PM
"It needs to include a discussion of the ValueExpression requirements"

### CODE REVIEW

File: getting-started/value-vs-data-binding.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ When using one-way binding, you *can* use the `<ParameterName>Changed` event. Fo
 
 To update the view-model in a one-way binding scenario, handle the corresponding `<Parameter>Changed` event. In the handler, update the view-model with the new data. This allows you to apply some synchronous logic to that new value and effectively cancel the update if you don't set the new value to the view-model.
 
+In some cases when using One-way binding, depending on the application structure you might get an error such as:
```
Comment (dimodi) [inline getting-started/value-vs-data-binding.md:43]:
```suggestion
In some cases when using one-way binding, depending on the application structure you might get an error such as:
```

File: getting-started/value-vs-data-binding.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ When using one-way binding, you *can* use the `<ParameterName>Changed` event. Fo
...
+
+>warning Error SystemInvalidOperationException: Telerik.Blazor.Components.SomeComponent requires a value for 'ValueExpression'. ValueExpression is provided automatically when using 'bind-Value'
+
+This error comes from the framework and applies to generic inputs as well, not only the Telerik components. To solve it, you can provide a `ValueExpression` to the component or use [Two-way binding](#two-way-binding). More details as well as an example you can find in [this knowledge base article]({%slug common-kb-requires-valueexpression%}).
```
Comment (dimodi) [inline getting-started/value-vs-data-binding.md:47]: * Mention Blazor framework to underline this is a generic issue.
* as well as = and
* "you can find" at that location is a Bulgarian word order
* better SEO for the KB link

```suggestion
This error comes from the Blazor framework and applies to generic inputs as well, not only the Telerik components. To solve it, you can provide a `ValueExpression` to the component or use [two-way binding](#two-way-binding). More details and an example are available in the [knowledge base article about required ValueExpression]({%slug common-kb-requires-valueexpression%}).
```

**Review by dimodi — APPROVED** [review]

**Review by marin-bratanov — COMMENTED** [review]
This is a duplicate of this earlier PR https://github.com/telerik/blazor-docs/pull/384/files

Consider keeping the first one so it is no so "in your face" about errors - IMO, the purpose of this article is to educate about concepts rather than to worry people with yellow blocks.

EDIT: You could add some more details to the paragraph I made (mine is, indeed, a tad too short), but I highly recommend shorteining the info about the error and removing the bright block.

**Comment by ntacheva** [comment]
Indeed, it is duplicate, apologies for the misunderstanding. I am now closing this one, approving and merging the other one. Let's keep the note short. Thus, it will be more likely for the clients to go ahead and check the kb for more details.

**Pull request closed without merging**